tests/test/linker-namespace-sanity/linker_namespace_base_test.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python2.7
+
+import os
+import sys
+
+
+sys.path.append(os.path.realpath(__file__ + '/../../../lib'))
+import udf
+
+
+class LinkerNamespaceBaseTest(udf.TestCase):
+
+    def setUp(self):
+        self.query('CREATE SCHEMA FN2', ignore_errors=True)
+        self.query('OPEN SCHEMA FN2')
+
+    def _execute_linker_namespace_udf(self, libs_to_check):
+        self.query(udf.fixindent('''
+                    CREATE OR REPLACE python3 SCALAR SCRIPT linker_namespace_test(search_string VARCHAR(100000)) RETURNS VARCHAR(100000) AS
+                    from ctypes import *
+                    from typing import List
+
+                    #Wrapper class for accessing libc function dl_iterate_phdr 
+                    #which is used to get all shared libraries in link-map (linker namespace)
+                    #See https://linux.die.net/man/3/dl_iterate_phdr for details of this function
+                    class SymbolScanner:
```

Comment:
I am thinking about, if we maybe should move this to an extra python file which we read and add to the create script via string operations.

Reply:
Hm, let's discuss offline.

Reply:
Done